src/main/java/green/citibike/json/StationInfo.java

```diff
@@ -8,4 +8,11 @@ public class StationInfo {
     public String name;
     @SerializedName("station_id")
```

Comment:
You should wait till I approve your PRs because this wasn't right.

Reply:
Okay, how should I fix it?

Reply:
Also I don't see that comment about checkstyles, can you repeat it?

Reply:
My mistake. I left that comment on someone else's PR. 
You can take out the `@SerializedName` and try wrapping the variable like this

``` java
    //CHECKSTYLE:OFF
    public String station_id;
    //CHECKSTYLE:ON
```

Tell me if that works.

Reply:
I don't think it did